modules/dirwatcher.js

```diff
@@ -0,0 +1,50 @@
+import EventEmitter from "events";
```

Comment:
Please use single quotes throughout the project. The modern JS styles tend to use 'single quotes' instead of "double quotes", even though actually they are the same thing.

Reply:
Resolved